### PR TITLE
AB#45 Implement Arena rendering

### PR DESCRIPTION
**Pull request by sir-marc**
- Add rendering for Units and Food
- Handle state updates from backend

### CODE REVIEW

**Comment by StuckiSimon** [comment]
Try to resize to 754px width and 587px height - there is a nice scrollbar flicker bug 😉

File: game-frontend/src/components/visual/canvas-wrapper.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
```
Comment (StuckiSimon) [inline game-frontend/src/components/visual/canvas-wrapper.jsx]: why is this `.jsx`?

**Comment by sir-marc** [comment]
> Try to resize to 754px width and 587px height - there is a nice scrollbar flicker bug 😉

Try another screen resolution then

**Review by michaelschufi — COMMENTED** [review]
This doesn't look clean. Could you scale it? We should also draw a grid imo?

image

**Comment by StuckiSimon** [comment]
> This doesn't look clean. Could you scale it? We should also draw a grid imo?
> 
> image

What do you mean with "clean"? If you mean the cluster of squares - this is due to the fact that each reload will give a new UserContext and therefore a new unit which gets spawned. Restart the backend to reduce the amount of units :v:

**Comment by michaelschufi** [comment]
No, it should be clear how many fields are between each squares. If you are starting out with the game, you should be able see clearly what the arena size is and which objects are on which field.

**Comment by michaelschufi** [comment]
Something like that?
image

**Review by StuckiSimon — APPROVED** [review]
Looks fine now.

**Pull request merged**